onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -90,6 +90,10 @@ namespace perftest {
         break;
       case 'x':
         test_config.run_config.enable_sequential_execution = false;
+        test_config.run_config.session_thread_pool_size = static_cast<int>(strtol(optarg, nullptr, 10));
+        if (test_config.run_config.session_thread_pool_size <= 0) {
```

Comment:
Can you use default value when it's <= 0? Previously, default value is 6.

Reply:
there is no such case. If `-x` is specified in commandline without a number after it, it will just print the help message and quit.